pymfl/api/MFLAPIClient.py

```diff
@@ -73,7 +72,8 @@ def _post(url: str, body: dict = None, **kwargs) -> dict | ET.Element:
         if body is None:
             body = dict()
         as_xml = kwargs.pop("as_xml")
-        response = requests.post(url, data=body)
+        session: Session = kwargs.pop("session")
```

Comment:
It seems session always required as a kwarg for this, should probably make it a required parameter for this method.